content/project/conduct.adoc

```diff
@@ -7,180 +7,246 @@ section: conduct
+* Commits
+* Pull requests
+* Issues
+* Wikis
 
+Technical criticism is always appreciated. Keep it positive and constructive. Don't merely decry the current state of affairs. Offer and solicit suggestions as to how things may be improved.
 (courtesy of the link:https://golang.org/conduct#values[Go Lang Code of Conduct])
 
 ==== Websites
 
-Everything hosted under link:https://jenkins-ci.org/[jenkins-ci.org] and its sub-domains such as:
+Everything hosted under link:https://jenkins.io/[jenkins.io], link:https://jenkins-ci.org/[jenkins-ci.org] and their sub-domains such as:
 
+* link:https://jenkins.io[jenkins.io]
 * link:https://issues.jenkins-ci.org/[issues.jenkins-ci.org]
 * link:https://wiki.jenkins.io/[wiki.jenkins.io]
-* link:http://lists.jenkins-ci.org/mailman/listinfo[lists.jenkins-ci.org]
 
 ==== Mailing lists
 
-Those hosted on Google Groups (see
-link:/mailing-lists[here]) and those self-hosted
-on link:http://lists.jenkins-ci.org/mailman/listinfo[lists.jenkins-ci.org].
-
-==== IRC Channels
+All link:/mailing-lists[mailing lists] hosted on Google Groups and other platforms.
 
-Documented in the
-link:/chat[IRC Channel] wiki
-page, e.g.
+==== Chats
 
-* `#jenkins`
-* `#jenkins-meeting`
-* `#jenkins-infra`
-* `#jenkins-community`
+* All chats documented on the link:/chat[this page], e.g. IRC channels: `#jenkins`, `#jenkins-meeting`, `#jenkins-infra`, etc.
+* Gitter channels within the link:https://gitter.im/jenkinsci/home[https://gitter.im/jenkinsci/] space
+* Jenkins chats hosted by Linux Foundation and its subsidiaries including Continuous Delivery Foundation 
 
 ==== Events
 
-* link:/projects/jam/[Jenkins
-  Area Meetup]
+* link:/projects/jam/[Jenkins Area Meetups]
+* link:/events/online-meetup[Jenkins Online Meetup]
 * Events where Jenkins is officially being represented (e.g.
   link:https://fosdem.org[FOSDEM], link:https://socallinuxexpo.org/[SCaLE])
-* Other Jenkins groups (such as conferences, meetups, and other unofficial
-  forums) are encouraged to adopt this Code of Conduct. Those groups must
-  provide their own moderators and/or reporting system.
 
+=== Other Jenkins Communities
+
+Other Jenkins groups (such as conferences, meetups, and other unofficial forums) and local communities are encouraged to adopt this Code of Conduct.
+Those groups must provide their own moderators and/or reporting system.
 
 == Reporting
 
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported by contacting the link:/project/board[Jenkins board].
 If you feel somebody has breached this code, please send an email with the
-relevant information (links, etc) to jenkinsci-board@googlegroups.com. This
-email list is only readable by the
-link:/project/board[Governance
-Board] members.
-
-Please understand that the board may not have all of the necessary context and
-history, so it's better to be more verbose than less verbose about such
-matters.
+relevant information (links, etc) to `jenkinsci-board@googlegroups.com`.
```

Comment:
Was this intentionally not a mailto link, even though CDF uses a mailto link in their contributor covenant?

I'm OK with it either way.  I like the highlighting gained by making it code font for greater visibililty.
```suggestion
relevant information (links, etc) to link:mailto:jenkinsci-board@googlegroups.com[`jenkinsci-board@googlegroups.com`].
```

Reply:
Mail to links are often not recommended  these days for use as many people don’t have their system mail client setup.

Reply:
Yes, I would prefer to not do it